### PR TITLE
GEOMESA-1260 Document GeoMesa Process deployment

### PR DESCRIPTION
**Pull request by jnh5y**
Signed-off-by: Jim Hughes <jnh5y@ccri.com>

### CODE REVIEW

**Comment by jnh5y** [comment]
@mdzimmerman, can you help me out with the formatting and a final read-through of this for the sense making?

File: geomesa-process/README.md [hunk 1]
```diff
@@ -3,15 +3,27 @@
...
\ No newline at end of file
+The above extensions are particular to the Accumulo datastore.
+
+While they can be used independently, the common use case is to use them with GeoServer.  To deploy them in GeoServer, one will require a) the GeoMesa Accumulo datastore plugin, b) the GeoServer WPS extension zip [], and c) the geomesa-process-${VERSION}.jar to be deployed in ${GEOSERVER_HOME}/WEB-INF/lib.  
+
+The GeoMesa Accumulo datastore plugin and GeoMesa process jars are both available in the binary distribution in the gs-plugins directory.
+
+Documentation about the GeoServer WPS Extension (including download instructions) is available here: http://docs.geoserver.org/2.8.1/user/extensions/wps/install.html.
+
+To verify the install, start GeoServer, and you should see a line like "INFO [geoserver.wps] - Found 11 bindable processes in GeoMesa Process Factory".
```
Comment (jahhulbert-ccri) [inline geomesa-process/README.md]: perhaps make the log line set inside of ticks so its monospace?

**Comment by jahhulbert-ccri** [comment]
gonna wait till 8am to see if @mdzimmerman looks over it else merging

File: geomesa-process/README.md (hunk 1 above)
Comment (mdzimmerman) [inline geomesa-process/README.md]: Or in a code block.

File: geomesa-process/README.md [hunk 2]
```diff
@@ -3,15 +3,27 @@
...
+
 ## Installation
 
-To install the GeoMesa processes, first deploy the main GeoMesa GeoServer plugin in 
- `${GEOSERVER_HOME}/WEB-INF/lib`.  Then copy the `geomesa-process-accumulo1.5-${VERSION}.jar` into the
- same directory.
\ No newline at end of file
+The above extensions are particular to the Accumulo datastore.
+
+While they can be used independently, the common use case is to use them with GeoServer.  To deploy them in GeoServer, one will require a) the GeoMesa Accumulo datastore plugin, b) the GeoServer WPS extension zip [], and c) the geomesa-process-${VERSION}.jar to be deployed in ${GEOSERVER_HOME}/WEB-INF/lib.
```
Comment (mdzimmerman) [inline geomesa-process/README.md]: [picky] I would put the JAR and file path in backticks.

Also, I am not sure what the empty square brackets in part B are for. Missing link?

**Pull request merged**